specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1368,6 +1368,379 @@
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The SQL database properties were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/SqlDatabaseListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      },
+      "put": {
+        "operationId": "DatabaseAccounts_CreateSqlDatabase",
+        "x-ms-examples": {
+          "CosmosDBSqlDatabaseCreate": { "$ref": "./examples/CosmosDBSqlDatabaseCreate.json" }
+        },
+        "description": "Creates an Azure Cosmos DB SQL database",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "createSqlDatabaseParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SqlDatabaseCreateUpdateParameters"
+            },
+            "description": "The parameters to provide for the current SQL database."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The SQL database create operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/SqlDatabase"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/apis/sql/databases/{databaseRid}": {
+      "get": {
+        "operationId": "DatabaseAccounts_GetSqlDatabase",
+        "x-ms-examples": {
+          "CosmosDBSqlDatabaseGet": { "$ref": "./examples/CosmosDBSqlDatabaseGet.json" }
+        },
+        "description": "Gets the SQL databases under an existing Azure Cosmos DB database account with the provided id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseRidParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The SQL database property was retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/SqlDatabase"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
Missing a PATCH api to support updates (required for tags, recommended for other mutable properties)

Reply:
We do not support PATCH, suppression was added in readme

Reply:
PATCH support for tags is mandatory, and is used by clients to update tags